web/gds-user-ui/src/components/Banner/LandingBanner.tsx

```diff
@@ -14,8 +14,8 @@ const LandingBanner = () => {
       <AlertIcon/>
       <AlertDescription fontWeight={'semibold'}>
         <Link href="https://calendar.app.google/FBg7GTmgDfeMbUMT9" isExternal>
-          <Trans id="Schedule a demo to learn about TRISA's open source self-hosted solution for cost-effective Travel Rule compliance.">
```

Comment:
The id isn't required to extract the text to a `.po` file for translation, so it has been remove here and in the other component file.